### PR TITLE
#6784 Microsoft Security Compliance Toolkit 1.0 list of PowerShell scripts

### LINKED ISSUE #6784: Scripts documentation missing
This main Security Compliance Toolkit page is missing documentation on the PowerShell scripts available in the Scripts folder of each baseline.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 014ca19c-6ab8-6eec-321e-72fa22c8f2c3
* Version Independent ID: 948a95a8-cb4f-668a-1988-17830d51845b
* Content: [Microsoft Security Compliance Toolkit 1.0 - Windows security](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-compliance-toolkit-10)
* Content Source: [windows/security/threat-protection/security-compliance-toolkit-10.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/security-compliance-toolkit-10.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @DulceMontemayor
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

Brian Steingraber requested the list of PowerShell scripts in Microsoft Security Compliance Toolkit 1.0 in issue ticket #6784 (**Scripts documentation missing**).

I also noticed that 2 of 3 links pointing to the Microsoft Security Baselines blog is still using the old and archived TechNet blog pages instead of the new and improved Tech Community pages, in addition to using the outdated name "Microsoft Security Guidance blog" instead of the new name "Microsoft Security Baselines blog".

**Changes proposed:**
- Add list of PowerShell script names, separated by file name
- Replace the outdated TechNet blog links with Tech Community links
- Update the link text names to "Microsoft Security Baselines blog".
- Remove redundant whitespace

**Ticket closure or reference:**

Closes #6784

### CODE REVIEW

**Comment by illfated** [comment]
#sign-off

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/security-compliance-toolkit-10.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The Policy Analyzer is a utility for analyzing and comparing sets of Group Polic
 
 Policy Analyzer lets you treat a set of GPOs as a single unit. This makes it easy to determine whether particular settings are duplicated across the GPOs or are set to conflicting values. Policy Analyzer also lets you capture a baseline and then compare it to a snapshot taken at a later time to identify changes anywhere across the set. 
 
-More information on the Policy Analyzer tool can be found on the [Microsoft Security Guidance blog](https://blogs.technet.microsoft.com/secguide/2016/01/22/new-tool-policy-analyzer/) or by [downloading the tool](https://www.microsoft.com/download/details.aspx?id=55319).
+More information on the Policy Analyzer tool can be found on the [Microsoft Security Baselines blog](https://techcommunity.microsoft.com/t5/microsoft-security-baselines/new-tool-policy-analyzer/ba-p/701049) or by [downloading the tool](https://www.microsoft.com/download/details.aspx?id=55319).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/security-compliance-toolkit-10.md:66]:
```suggestion
More information on the Policy Analyzer tool can be found in the [Microsoft Security Baselines blog](https://techcommunity.microsoft.com/t5/microsoft-security-baselines/new-tool-policy-analyzer/ba-p/701049) or by [downloading the tool](https://www.microsoft.com/download/details.aspx?id=55319).
```

**Comment by mypil** [comment]
@DulceMontemayor - PR has been copy edited and is ready for final review. This is based on issue #6784.

Thank you.

cc: @kenwith

**Comment by illfated** [comment]
Please note that this PR does not resolve the referenced issue ticket, it is merely a preparation to make it easier moving forward with adding any of the requested documentation for scripts (whether it will be links to existing documentation or adding new documentation, I don't know yet).

**Comment by illfated** [comment]
1 month later, still no sign of feedback from anyone with merging rights. Is this delay due to how the COVID-19 status in the US affects the number of available employees to perform their regular work? Or is there something else causing this PR to be put on a low priority list or on hold?

**Comment by illfated** [comment]
I wonder if this page needs a full rewrite, because it seems to be somewhat clunky (difficult to be more specific).

Then again, I presume there may just as well be plans for writing a page named something like **Microsoft Security Compliance Toolkit 2.0** , so in that case, my idea is not very useful.

**Comment by illfated** [comment]
(Ref. #7073 )

**Pull request merged**